pkgs/by-name/ir/irtt/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "irtt";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "heistp";
+    repo = "irtt";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-b0Od/KWV408PP/rPtqMr+lTyPXlxigLEU2/by6gjfxg=";
+  };
+
+  vendorHash = "sha256-Mxdvb1d44dag1/jKjSVAzqYBed/lylyamgwQwj9THl4=";
+
```

Comment:
```nix
  nativeInstallCheckInputs = [ versionCheckHook ];
  versionCheckProgramArg = "version";
  doInstallCheck = true;
```

Reply:
What do those do?

Reply:
The install check phase runs after the install phase. For this particular setup, it will run `irtt version` and check that it finishes with zero exit code and contains the package version in the stdout.